python/kvikio/pyproject.toml

```diff
@@ -127,6 +127,7 @@ ninja.make-fallback = true
 sdist.exclude = ["*tests*"]
 sdist.reproducible = true
 wheel.packages = ["kvikio"]
+wheel.exclude = ["*.pyx", "*.pxd"]
```

Comment:
oh nice! Makes sense to me that `kivkio` wouldn't need to include these sources, and that`libkvikio` would be the only one that needs the library and headers.

This is a good trick to keep in mind as we continue with other libraries for https://github.com/rapidsai/build-planning/issues/33.

Reply:
I agree with not including pyx files, but we should be careful about not including pxd files. I _think_ this is currently fine for kvikio because none of kvikio's pxd files are intended for public consumption, but that is definitely not going to be generally true in RAPIDS. Most of the time we do want to include the pxd files.

Reply:
I've put back the `*.pxd` files.

Reply:
I've also opened https://github.com/rapidsai/ucxx/pull/270 to correct a mistake I made.